qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -907,6 +908,64 @@ def autorange_once(self) -> None:
         self.write('SENSe:VOLTage:DC:RANGe:AUTO ONCE')
         self.range.get()
 
+    def increase_range(
+        self,
+        range_value: Optional[Union[float, int]] = None,
```

Comment:
This can be simplified to `Optional[float]` the hierarchy of numeric types in python is such that a float value is allowed to be an int too

Reply:
That makes sense.  I'll change that.